production/helm/loki/templates/index-gateway/statefulset-index-gateway.yaml

```
@@ -71,6 +71,10 @@ spec:
         - name: index-gateway
           image: {{ include "loki.image" . }}
           imagePullPolicy: {{ .Values.loki.image.pullPolicy }}
+          {{- if or .Values.loki.command .Values.indexGateway.command }}
+          command:
+            - {{ coalesce .Values.indexGateway.command .Values.loki.command | quote }}
+          {{- end }}
```

What did you think about the following syntax? it would also support multiple command, as Kubernetes support it, which introducing a breaking change

```suggestion
          {{- with coalesce .Values.loki.command .Values.indexGateway.command }}
          command:
            {{- if kindIs "slice" . }}
            {{- toYaml . | nindent 2 }}
            {{- else }}
            - {{ . | quote }}
            {{- end }}
          {{- end }}
```

---

Does it make sense to support two different variable formats? Will it not make it hard to maintain, or even lead to human errors on the variable definition?

---

If yes, do you agree to make this standard and retro change this support in all manifests to be equal everywhere?